### ISSUE TITLE
Chip Ram access slow 

### ISSUE THREAD

**Issue by radi7777** [michalsc/Emu68#155]
Hi, looks like programs starting lower than on regular 68030..
some games are slower than on stock a500...
when you run an Hires interlace mode 8 colors on a RGB monitor the system feels like glue ...
looks like adressing chipram is slow.... is this a bug or is it just how it is ?? bekaus of limitations of Pistorm ?

**Comment by michalsc** [comment]
Please add some more information. Kickstart version, workbench version. Are you using SD card as hard drive or some other solution? I need more information.

Please keep in mind chip ram is in general relatively slow on current PiStorm version (about 2MB/s) so maybe this is what you see.

**Comment by radi7777** [comment]
Hi Michael, Im running an Amiga500 rev5 with 1Mb Chip / Kick 3.2 eprom 
Workbench 3.2.  As harddrive Im using the Pi 3a+ SD card ( 128GB) 
As kick.rom im using 3.2.1 on the SD card.

I think as long as the Chip ram is slow its really a problem for Games... ( WhDload 98 % are fine ) but disk versions are really a problem ...


Thanx for your hard work.. really awesome project...

is there any chance to fix chipramspeed at all???  maybe pistorm isnt the right Base for your project

**Comment by captain-amygdala** [comment]
@radi7777 
Yes, there is a chance chipram speed will be fixed. But currently I have too much on my todo list for a immediately fix.
As intermediate solution you can overclock the Raspberry Pi "Core" frequency and use the experimental PiStorm Firmware.
This reports 96% to 100% Chipram Speed in Sysinfo.

**Comment by elerion64** [comment]
@captain-amygdala
Tried overclocking RPi3A to 1500MHz, but that even further reduced Chip speed. (From 0,96 to 0,93 in Sysinfo.) Where can we find this experimental Firmware?
